src/components/PineTypes/Boolean.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react'
 import Select from '../Select'
-import * as isBoolean from 'lodash/isBoolean'
-import * as isString from 'lodash/isString'
+const isBoolean = require('lodash/isBoolean')
+const isString = require('lodash/isString')
```

Comment:
Why do we need this?

Reply:
I think jest was choking on the import syntax here